test/components/views/rooms/MessageComposerButtons-test.tsx

```diff
@@ -115,6 +119,56 @@ describe("MessageComposerButtons", () => {
             ],
         ]);
     });
+
+    describe('polls button', () => {
```

Comment:
I recognize this test is a bit weird, but it's probably better than nothing. Testing that the `MessageComposer` responds to the settings store feels a bit weird, so haven't written the test. Happy to be convinced otherwise though.